os/src/main/kotlin/com/gojuno/commander/os/Processes.kt

```diff
@@ -61,10 +61,11 @@ fun process(
             }
 
             val process: Process = ProcessBuilder(command)
+                    .redirectErrorStream(true)
                     .let {
                         when (unbufferedOutput) {
-                            true -> it
-                            else -> it.redirectErrorStream(true).redirectOutput(ProcessBuilder.Redirect.to(outputFile))
+                            true -> it.redirectOutput(File("/dev/null"))
```

Comment:
Let's add a function to get OS-dependent `/dev/null` `File`?

On Windows it should be `NUL` according to this SO https://stackoverflow.com/questions/313111/dev-null-in-windows